addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/test/YamahaEmulatedHttpXMLServer.java

```diff
@@ -0,0 +1,286 @@
+package org.openhab.binding.yamahareceiver.test;
+
```

Comment:
It seems that this file has a missing  or invalid copyright header, can you fix all of them by running 

`mvn license:format` 

Please commit only the ones for your binding.